### PR TITLE
Fix failing unit-tests due to fda.usc update

### PR DESCRIPTION
**Pull request by pfistfl**
This should fix the failing unittests due to the CRAN update of package `fda.usc` to version 1.4
See #2173 for details.

I additionally added a seed, as the benchmark would throw warnings if values in the test set are not in the range of values in the training set.

### CODE REVIEW

**Comment by larskotthoff** [comment]
Thanks, merging.

**Comment by vrodriguezf** [comment]
This is failing for me again, when executing `expect_error(train(lrn, subsetTask(tsk, features = 1:3)), "numeric inputs")`

````{r}
test_that("FDA properties work", {
  df = data.frame(matrix(rnorm(100), nrow = 10), "target" = sample(letters[1:2], 10, replace = TRUE))
  # Transform to functional data (fd.features indicate column names or indices)
  fdf = makeFunctionalData(df, fd.features = list("fd1" = 1:6, "fd2" = 8:10))
  # Create a classification task
  tsk = makeClassifTask(data = fdf, target = "target")
  lrn = makeLearner("classif.fdausc.knn")
  # Error for multiple functional inputs
  expect_error(resample(lrn, subsetTask(tsk, features = 2:3), cv2), "more than one functional inputs")
  # Error for numeric inputs
  expect_error(train(lrn, subsetTask(tsk, features = 1:3)), "numeric inputs")
  expect_error(train(lrn, subsetTask(tsk, features = 1)), "numeric inputs")
  # No error for single functional
  # FIXME Undoc below with fda.usc update
  # expect_silent(train(lrn, subsetTask(tsk, features = 2)))
  # expect_silent(train(lrn, subsetTask(tsk, features = 3)))
})
```
